### PR TITLE
Allow specifying purs version via env var

### PR DESCRIPTION
**Pull request by drewolson**
If no version is specified, the latest is installed.

### CODE REVIEW

File: install.js [hunk 1]
```diff
@@ -1,13 +1,18 @@
 "use strict";
 
+const process = require("process");
 const https = require("follow-redirects").https;
 const tar = require("tar");
 const shell = require("shelljs");
-const version = require("./package.json")["purs-version"];
+const version = process.env.PURS_VERSION;
```
Comment (justinwoo) [inline install.js]: should this `|| require("./package.json")["purs-version"]` right?

File: README.md [hunk 2]
```diff
@@ -20,11 +20,10 @@ You can also install this globally:
...
 
-## Version history
+You can specify a version using the `PURS_VERSION` env var:
 
-Instead of awkwardly trying to follow the same versioning scheme, the following table should be consulted if you need a specific series of versions:
+```
+PURS_VERSION=v0.13.0 npm install -g purs-bin-simple
+```
 
-| npm package | purs        |
```
Comment (justinwoo) [inline README.md]: i think it's good to keep this table, and just say that this will be used.

File: install.js [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
 const platform = { win32: "win64", darwin: "macos" }[process.platform] || "linux64";
+const urlPrefix = "https://github.com/purescript/purescript/releases";
+const url = version !== undefined
+  ? `${urlPrefix}/download/${version}/${platform}.tar.gz`
+  : `${urlPrefix}/latest/download/${platform}.tar.gz`
```
Comment (justinwoo) [inline install.js]: i dont know if this is so good, since this means that there's no guarantee of a stable version being installed

**Comment by drewolson** [comment]
@justinwoo updated to address your concerns.

**Comment by justinwoo** [comment]
Cool, thanks

**Pull request merged**